### PR TITLE
Services

### PR DESCRIPTION
**Pull request by TripppleG**

### CODE REVIEW

File: src/main/java/com/flightbookingsystem/services/implementations/AirportServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
+public class AirportServiceImpl implements AirportService {
+    private final AirportRepository аirportRepository;
```
Comment (kpetrova3) [inline src/main/java/com/flightbookingsystem/services/implementations/AirportServiceImpl.java]: How can we use AirportRepository when it does not exist

File: src/main/java/com/flightbookingsystem/services/implementations/CityServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
+public class CityServiceImpl implements CityService {
+    private final CityRepository cityRepository;
```
Comment (kpetrova3) [inline src/main/java/com/flightbookingsystem/services/implementations/CityServiceImpl.java:23]: not existing CityRepository

**Review by kpetrova3 — COMMENTED** [review]
Abstractions are used that do not exist in the code

File: src/main/java/com/flightbookingsystem/services/implementations/CityServiceImpl.java (hunk 2 above)
Comment (TripppleG) [reply src/main/java/com/flightbookingsystem/services/implementations/CityServiceImpl.java:23]: We will create them. They are essential

**Review by kpetrova3 — APPROVED** [review]

**Pull request merged**